runtime/moonbase/src/lib.rs

```diff
@@ -199,7 +199,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 2,
-	state_version: 0,
+	state_version: 1,
```

Comment:
The benchmarks will still need to account for the costs of State V0. 
Since `benchmark pallet` does not have a `--state-version` argument, we will need something like the suggestion below while the state is not completely migrated.

```suggestion
	#[cfg(feature = "runtime-benchmarks")]
	state_version: 0,
	#[cfg(not(feature = "runtime-benchmarks"))]
	state_version: 1,
```